swap/protocol.go

```diff
@@ -76,13 +75,13 @@ func (s *Swap) APIs() []rpc.API {
 
 // Start is a node.Service interface method
 func (s *Swap) Start(server *p2p.Server) error {
-	log.Info("Swap service started")
+	s.logger.Info("Swap service started")
```

Comment:
However, I would keep this stuff in the normal log. It doesn't contain financial information.

Reply:
This was a concern that we discussed with @mortelli 
I could redefine the flag to 
> debugswapfile

and let the rest of the code continue logging onto the standard output.
The only concern is if you have other packages that need also to log to the same output, you would need multiple flags.

And also for debugging purposes, you have to search in the output and also in the log files.
That's why I unified the logs.

But I can change it back only to log in the swap package and ask core if this would be nice to add.

Definitely not in this PR I suppose.

Reply:
if we look at the issue that this PR solves, we are aiming to have a "swap logger". 

based on this, i think it's convenient to have this logger include the entry for the swap service being started.

with this message we could determine, for example—for the case of no accounting activity—whether swap is being started but not doing anything or if it's not even being started at all.

Reply:
I also think that this message belongs to the `swap` logger.

Reply:
I have mixed feeling on this one.
As it only indicates the start and stop of swap it may be ok to leave it in the swarm log.
If the swap could be enabled/disabled on runtime, then I would suggest including this log into the swap logger as well.